### PR TITLE
PP-6375 Add CI command

### PR DESCRIPTION
**Pull request by danworth**
Has sub-command `compare` which compares the outcome of ci builds
between jenkins and concourse.

Filters out results prior to enabling Concourse on the 27/4/2020.

with @rorymalcolm

### CODE REVIEW

**Review by rorymalcolm — DISMISSED** [review]
nice work !!!

**Comment by rorymalcolm** [comment]
actually - this doesn't work for e2e atm unless you have an access token with elevated permissions

File: pkg/cmd/root.go [hunk 1]
```diff
@@ -27,6 +27,7 @@ func Execute() {
 		API(),
 		Card(),
 		Toolbox(),
+    CI(),
```
Comment (sfount) [inline pkg/cmd/root.go]: Very minor but the spacing on this doesn't look happy

File: pkg/ci/compareWithJenkins.go [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+		return "no build"
+	}
+
+	return state
+}
+
+func initClient() {
+	var tokenClient *http.Client
+	oauthToken := os.Getenv("PAY_CLI_GITHUB_ACCESS_TOKEN")
+	if oauthToken != "" {
```
Comment (sfount) [inline pkg/ci/compareWithJenkins.go:127]: Nice 👌

**Review by sfount — DISMISSED** [review]

File: pkg/cmd/root.go (hunk 1 above)
Comment (rorymalcolm) [reply pkg/cmd/root.go]: pressed CTRL+S and fixed this!

**Review by rorymalcolm — DISMISSED** [review]
👍

**Review by rorymalcolm — APPROVED** [review]
`gofmt`

**Pull request merged**